### PR TITLE
[4.0] Legacy cleanup plg_system_remember

### PR DESCRIPTION
**Pull request by HLeithner**
### Summary of Changes
Cleaning up legacy code.

### Testing Instructions
Test it in combination with the prepared statement PR #25539.
Use remember me function.

### Expected result
Nothing changed.

### CODE REVIEW

File: plugins/system/remember/remember.php [hunk 1]
```diff
@@ -42,29 +42,17 @@ class PlgSystemRemember extends CMSPlugin
 	 */
 	public function onAfterInitialise()
 	{
-		// Get the application if not done by JPlugin. This may happen during upgrades from Joomla 2.5.
```
Comment (brianteeman) [inline plugins/system/remember/remember.php:45]: will someone have to upgrade to 3 before 4? if they can upgrade from 2.5 directly then we still need this

Comment (HLeithner) [reply plugins/system/remember/remember.php:45]: You have to upgrade to 3.10 first.

Comment (brianteeman) [reply plugins/system/remember/remember.php:45]: ok

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**